arbeitszeit_flask/templates/company/invite_worker_to_company.html

```diff
@@ -46,11 +46,15 @@ <h1 class="title">{{ gettext("Workers of the company") }}</h1>
             {% for worker in view_model.workers %}
             <tr>
               <td>{{ worker.id }}</td>
-              <td>{{ worker.name}}</td>
+              <td>{{ worker.name}}</td>                
             </tr>
             {% endfor %}
           </table>
         </div>
+        <h1 class="title">{{ gettext("Remove workers") }}</h1>
+        <div>{{ gettext("Visit this link to remove a worker:") }}
```

Comment:
s/Visit this link/Click here/  (? — One does not actually visit a link, one visits a page by following a link.  I know, tiny and fussy point.)